### DISCUSSION TITLE
Pls help how to reset ant media password of community shared cpu

### DISCUSSION THREAD

**Discussion by jasvinder2020** [ant-media/Ant-Media-Server#6029] [Q&A]
Hi 

Pls help how to receive antmedia password hosted on shared cpu through Linode

**Comment by USAMAWIZARD** [comment]
Stop the server using `service antmedia stop  `
Delete the server.db file  `rm -rf /usr/local/antmedia/server.db`
Start the server using `service antmedia start`
